### PR TITLE
Move Laravel image from PHP variant to top-level image

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR migrates the `php:latest-laravel` and `php:latest-laravel-dev` variants as top-level images, so that they have a dedicated documentation and landing pages that will facilitate devRel activities around the Laravel community. This was discussed in Slack with @amdawson FYI.

### Image Size

- [X] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

### Image Vulnerabilities

- [ ] The Grype vulnerability scan returned 0 CVE(s).
- [X] The Grype vulnerability scan returned > 0 CVE(s) (please explain in the notes).

Notes: this image is based on our `php:latest-fpm` image. They both have 1 medium CVE:
```
NAME         INSTALLED  FIXED-IN  TYPE  VULNERABILITY  SEVERITY 
php-8.2-fpm  8.2.18-r0            apk   CVE-2015-3211  Medium
```

### Image Tagging

- [ ] The image is _not_ tagged with version tags.
- [X] The image is tagged with :latest
- [ ] The image is not tagged with :latest (please explain in the notes).

Notes:

### Basic Testing - K8s cluster

- [ ] The container image was successfully loaded into a kind cluster.
- [ ] The container image could not be loaded into a kind cluster (please explain in the notes).

Notes: I don't have access to this infrastructure.

### Basic Testing - Package/Application

- [X] The application is accessible to the user/cluster/etc. after start-up.
- [ ] The application is not accessible to the user/cluster/etc. after start-up. (please explain in the notes).

Notes:

### Helm

- [ ] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.
- [ ] A Helm chart has been provided and the container image is not working with the chart (please explain in the notes).
- [X] A Helm chart was not provided.

Notes:

### Processor Architectures

- [X] The image was built and tested for x86_64.
- [ ] The image could not be built for x86_64 (please explain in the notes).
- [ ] The image was built and tested for aarch64.
- [X] The image could not be built for aarch64. (please explain in the notes).

Notes: I did not make this test because the image is not new, it's just moving places

### Functional Testing + Documentation

- [X] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes:

### Environment Testing + Documentation

- [ ] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.
- [ ] The container image has been tested successfully on a public cloud provider (AWS, GCP, Azure).
- [X] The container image has not been tested successfully on a public cloud provider (AWS, GCP, Azure) (please explain in the notes).

Notes: I have only tried it locally standalone and with Docker Compose

### Version

- [X] The package version is the latest version of the package.  The latest tag points to this version.
- [ ] The package version is the not the latest version of the package (please explain in the notes).

Notes:

### Dev Tag Availability

- [X] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ] There is not a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base') (please explain in the notes).

Notes:

### Access Control + Authentication

- [ ] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
- [X] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [X] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).

Notes: The image includes a **laravel** user designed for development, but it's not the default user running commands. Default user is **php** with uid `65532` named like that to comply with common PHP images.

### ENTRYPOINT

- [X] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]
- [ ] applications/servers/utilities not set to call main program with no arguments e.g. [redis-server] (please explain in the notes)
- [ ] base images leave empty.
- [ ] base image and not empty (please explain in the notes).
- [ ] dev variants is set to entrypoint script that falls back to system.
- [ ] dev variants is not set to entrypoint script that falls back to system (please explain in the notes).

### CMD

- [X] For server applications give arguments to start in daemon mode (may be empty)
- [ ] For utilities/tooling bring up help e.g. `–help`
- [ ] For base images with a shell, call it e.g. [/bin/sh]

### Environment Variables

- [ ] Environment variables added.
- [X] Environment variables not added and not required.

### SIGTERM

- [ ] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [X] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [X] A README file has been provided and it follows the README template.

### CODE REVIEW

File: images/php/README.md [hunk 1]
```diff
@@ -133,6 +142,12 @@ For more detailed information on how to use these images, check the [Getting Sta
 
 ## Detailed Environment Information
 
+To obtain information about available modules, you can run:
+
+```shell
+docker run --rm --entrypoint php cgr.dev/chainguard/laravel-latest -m
```
Comment (jonjohnsonjr) [inline images/php/README.md:148]:
```suggestion
docker run --rm --entrypoint php cgr.dev/chainguard/laravel -m
```

File: images/php/README.md [hunk 2]
```diff
@@ -41,14 +40,24 @@ This will automatically pull the image to your local system and execute the comm
...
 Zend Engine v4.2.1, Copyright (c) Zend Technologies
 ```
 
+## Running Composer
+
+To install application dependencies from your host machine, you can use the `latest-dev` variant with a shared volume:
+
+```shell
+docker run --rm -v ${PWD}:/work --entrypoint composer --user root \
+    cgr.dev/chainguard/php-dev \
```
Comment (jonjohnsonjr) [inline images/php/README.md:57]: Is this right?

Comment (erikaheidi) [reply images/php/README.md:57]: oops, good catch. fixing it now

**Review by jamonation — APPROVED** [review]
Built locally and followed the tutorial, works great!
```
docker run -p 8000:8000 --rm -it -v ${PWD}:/work --entrypoint /work/demo-laravel/artisan --user laravel  k3d-k3d.localhost:5005/laravel:latest-dev serve --host=0.0.0.0

   INFO  Server running on [http://0.0.0.0:8000].
```

**Pull request merged**